tests/fs-test.rs

```diff
@@ -1,24 +1,26 @@
 mod common;
 
 use std::{
-	fs::{read, remove_file},
+	fs::{read_to_string, remove_file},
 	path::PathBuf,
 };
 
-use common::{build_hermit_bin, run_simple_vm};
+use common::{build_hermit_bin, remove_file_if_exists, run_simple_vm};
+
+/// Verifies successful file creation on the host OS and its contents.
+pub fn verify_file_contents(testfile: &PathBuf, contents: &str) {
```

Comment:
I'd call this function `verfiy_file_equals` or similar, as I can imagine we need a function `verify_file_contains` in the future as well.